### PR TITLE
filter PAN when describe message

### PR DESCRIPTION
**Pull request by alovak**

### CODE REVIEW

File: describe.go [hunk 1]
```diff
@@ -13,6 +14,13 @@ import (
...
 var defaultSpecName = "ISO 8583"
 
+var panFilter = regexp.MustCompile(`^\d(\d*)\d{4}$`)
+
+const (
+	bitmapIndex = 1
+	panIndex    = 2
+)
+
 func Describe(message *Message, w io.Writer) error {
```
Comment (adamdecaf) [inline describe.go:24]: FYI human-readable output typically isn't protected from breaking changes, so we're free to alter this as needed. Hiding PII is very important. 

We offer multiple machine readable formats which we promise to not break.

**Comment by wadearnold** [comment]
Does this seem like it could get merged to filter PII data? What do you think @adamdecaf @alovak

File: describe_test.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ MTI...........................: 0100
 Bitmap........................: 400000000000000000000000000000000000000000000000
 Bitmap bits...................: 01000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 F000 Message Type Indicator...: 0100
-F002 Primary Account Number...: 4242424242424242
+F002 Primary Account Number...: 4****4242
```
Comment (adamdecaf) [inline describe_test.go:51]: I think the output should match the original length. PANs are pretty known lengths, so we aren't revealing much by showing the actual length.

**Comment by alovak** [comment]
Implemented in https://github.com/moov-io/iso8583/pull/203

**Pull request closed without merging**